README.md

```diff
@@ -162,5 +162,6 @@ See [example usage](docs/alignment-toolbar.md#example-usage--es5-) of alignment-
 #### Example Usage:
 
 - See [examples](examples/)
+	* To enable example blocks, Keep `IS_GUTENBERG_FIELDS_MIDDLEWARE_DEV` constant true. 
```

Comment:
Lets change this name to `GUTENBERG_FIELDS_MIDDLEWARE_IS_DEV` and add instruction  
"You can set `GUTENBERG_FIELDS_MIDDLEWARE_IS_DEV` to `true` during development in `wp-config.php` which will also enable example blocks."

Reply:
@sayedtaqui Made changes as suggested.